fabric/patches/3_0001-Add-FPC-validator-to-library.patch

```diff
@@ -9,25 +9,32 @@ peer.
 
 Signed-off-by: Marcus Brandenburger <bur@zurich.ibm.com>
 ---
- core/handlers/library/fpc_lib_handler.go | 23 +++++++++++++++++++++++
- 1 file changed, 23 insertions(+)
+ core/handlers/library/fpc_lib_handler.go | 30 ++++++++++++++++++++++++
+ 1 file changed, 30 insertions(+)
  create mode 100644 core/handlers/library/fpc_lib_handler.go
 
 diff --git a/core/handlers/library/fpc_lib_handler.go b/core/handlers/library/fpc_lib_handler.go
 new file mode 100644
-index 000000000..92a355522
+index 000000000..850990746
 --- /dev/null
 +++ b/core/handlers/library/fpc_lib_handler.go
-@@ -0,0 +1,23 @@
+@@ -0,0 +1,30 @@
 +//+build FPC
 +
++/*
++Copyright IBM Corp. All Rights Reserved.
++Copyright 2020 Intel Corporation
++
++SPDX-License-Identifier: Apache-2.0
++*/
++
 +package library
 +
 +import (
-+	validation "github.com/hyperledger/fabric/core/handlers/validation/api"
-+	decoration "github.com/hyperledger/fabric/core/handlers/decoration"
-+	"github.com/hyperledger-labs/fabric-private-chaincode/validation/fpc"
 +	"github.com/hyperledger-labs/fabric-private-chaincode/validation/ercc"
++	"github.com/hyperledger-labs/fabric-private-chaincode/validation/fpc"
++	decoration "github.com/hyperledger/fabric/core/handlers/decoration"
```

Comment:
```suggestion
+	"github.com/hyperledger/fabric/core/handlers/decoration"
```

Should also work like that right?

Reply:
maybe, only reason i touched that file (and only thing i did) is to make it properly formatted to pass `make linter`. Would prefer not changing other change anything else in this pr ..

Reply:
Sometimes the formatter/importer programs does this duplicative thing, I don't know why.